### PR TITLE
Allow the user to disable print statements

### PR DESCRIPTION
**Pull request by zurgeg**
This one is self-explanatory :)

### CODE REVIEW

File: p3dopenvr/p3dopenvr.py [hunk 1]
```diff
@@ -22,8 +22,9 @@ def exitfunc():
     openvr.shutdown()
 
 class P3DOpenVR():
-    def __init__(self):
+    def __init__(self,verbose):
```
Comment (el-dee) [inline p3dopenvr/p3dopenvr.py]: Could you add a space after the comma and set the default value for verbose to True

**Review by el-dee — CHANGES_REQUESTED** [review]
Thanks for this, just one small remark to fix and I'll merge it.

**Comment by zurgeg** [comment]
Yep! Done @el-dee!

**Review by el-dee — APPROVED** [review]

**Comment by el-dee** [comment]
Thanks, merged in https://github.com/el-dee/panda3d-openvr/commit/7957473da6efcc8a96760555691ba89e9186654d

**Pull request closed without merging**